packages/io/amf-deserializer/.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
```

Comment:
this is not needed, nor will it work: there is only one git repo now , the root one :)

Reply:
Done